server/routes/serviceProviderRoutes.ts

```diff
@@ -17,7 +17,18 @@ export default function serviceProviderRoutes(router: Router, services: Services
     services.referenceDataService
   )
 
-  get(router, '/dashboard', (req, res) => serviceProviderReferralsController.showDashboard(req, res))
+  get(router, '/dashboard', (req, res) => serviceProviderReferralsController.showMyCasesDashboard(req, res))
```

Comment:
I wonder if it would make sense to make this a redirect instead to reduce future bookmarking (maybe?):
```
  get(router, '/dashboard', (req, res) => res.redirect(301, `${serviceProviderUrlPrefix}/dashboard/my-cases`))
```

It would probably allow us to make the test for this smaller, testing that it's redirected, rather than duplicating the two tests.

Up to you though!